integration-tests/src/test/java/oracle/weblogic/kubernetes/ItRecoveryDomainInPV.java

```diff
@@ -132,7 +132,7 @@ public static void initAll(@Namespaces(2) List<String> namespaces) {
    * Make sure all 100 persistent messages are recovered form managed server(2)
    */
   @Test
-  @DisplayName("verifies persistent WebLogic data survies the server pod")
+  @DisplayName("verifies persistent WebLogic data survives the server pod")
```

Comment:
Change the description as follow ..
    @DisplayName("verifies persistent WebLogic data survives the server pod scaling")